src/Panels/Middlewares/view.html

```diff
@@ -0,0 +1,13 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
```

Comment:
Is there any opportunity not to use jquery?

Reply:
Not yet. But I'll do it soon without jquery.